src/index.html

```diff
@@ -11,6 +11,28 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-form"
+      data-qa="big"
+    >
+      <input
+        class="search-input big-form"
+        data-qa="keypress"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+
+    <form
+      class="search-form"
+      data-qa="small"
+    >
+      <input
+        class="search-input small-form"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+
   </body>
```

Comment:
```suggestion
    </form>
  </body>
```
Don't leave empty lines between parent and child elements